### ISSUE TITLE
Recreate the vendor profiles as extended HL7 Sweden Base Profiles

### ISSUE THREAD

**Issue by vjernelov** [commonprofiles-care/fhir#21]
Now that HL7 Sweden have balloted the following profiles:

- Organization
- Patient
- Practitioner
- PractitionerRole

We should technically extend our profiles from these.

Source: [http://hl7.se/fhir/ig/base/profiles.html](http://hl7.se/fhir/ig/base/profiles.html)

**Comment by vjernelov** [comment]
@peturv starts with the Practitioner profile.

If we find that something doesn't work we can use this as a source of feedback to HL7 Sweden.

Organization will need to wait since an issue with the binding to the Organization.code has been reported to HL7 Sweden.

**Comment by niclas86** [comment]
@peturv did you do this? We can´t see that its correct in the implementation guide

**Comment by niclas86** [comment]
@peturv will check with Arvid about dependency and will create a pull request after

**Comment by peturv** [comment]
During alignment of practitioner profile I discovered a discrepancy of OID for HSA-ID.
In our profile we had defined `1.2.752.129.2.1.4.1`, in HL7SE `1.2.752.29.4.19` was used.

From what I can see, both series are managed to Inera, directly or indirectly:
1.2.752.29 = Inera AB / Inera AB (previously, SPRI) Allocated by KTHNOC
1.2.752.129 = Sveriges Kommuner och landsting / The OID shall be used within ongoing computerization for social welfare and health care.

According to recently updated information at Inera, `1.2.752.129.2.1.4.1` should be used for objects in the HSA-catalogue: https://inera.atlassian.net/wiki/spaces/KINT/pages/468746902/Identifierare+p+Inera

But according to HSA-schema organisationsträdet 4.18.2 `1.2.752.29.4.19` is used for HSAId : https://inera.atlassian.net/wiki/spaces/OIKH/pages/2658271294/HSA-schema+version+4.18.2+-+g+ller+t.o.m.+2022-12-07 

While this is not an ambiguity in our defintions, it is definately something that affects us. We need to request clarification, not only towards us but alse in communication from Ineras part.
